LiskHQ/lisk-desktop
# Bug fix: verify routes on deep linking navigation

### What was the problem?

This PR resolves #5117

### How was it solved?

- whitelist and verify route on `open-url` event 

### How was it tested?

- Run the electron application `yarn build:electron && yarn start`
- Add a console.log within the open-url event in `app/src/main.js` in order to observe the response from `canExecuteDeepLinking`
- Open up another terminal and run `open "lisk://wallet/?modal=send&recipient=lskdb3ja5dhr25jhwm6onstawj33yk74g8q2549oc&amount=1&token=0200000000000000&recipientChain=02000000"`
- `canExecuteDeepLinking` should return true
- Try different combinations of deep links to test durability of the deep link verification logic
